Platform/TigerlakeBoardPkg/AcpiTables/SsdtRtd3/Rtd3Cnvi.asl

```diff
@@ -0,0 +1,77 @@
+/** @file
+  Intel ACPI sample code for Connectivity devices

```

Comment:
I think it is not necessary to add this file for S0ix support.

Reply:
Done removed it, thanks.